components/identity.oauth.uma.permission.service/src/main/java/org/wso2/carbon/identity/oauth/uma/permission/service/ReadPropertiesFile.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.uma.permission.service;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.oauth.uma.permission.service.model.PermissionTicketDO;
+import org.wso2.carbon.utils.CarbonUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+/**
+ * ReadPropertiesFile is used to read configuration values from a properties file which is
+ *required for a permission ticket generation.
+ */
+//TODO: Move this configuration to identity.xml
```

Comment:
Remove the todo. If there is any improvement needs to be done, please report an issue.

Reply:
Removed reading configuration values from a properties file in https://github.com/wso2-extensions/identity-oauth-uma/pull/22/files#diff-b690bd19be52253885c067b5654785e0L38